common/lib/capa/capa/inputtypes.py

```diff
@@ -629,8 +629,11 @@ def get_attributes(cls):
                                              #   set state
             Attribute('width', "400"),       # iframe width
             Attribute('height', "300"),      # iframe height
-            Attribute('sop', None)           # SOP will be relaxed only if this
-                                             # attribute is set to false.
+            # Title for the iframe, should be supplied by the author of the problem. Not translated
+            # because default display name is also not translated, and we can't import Django here anyway.
+            Attribute('title', "Custom Javascript Display Iframe"),
```

Comment:
@catong suggestions for a better default title? The current default display name for this component is "Custom Javascript Display and Grading".